Libraries/Tile/Tile.h

```diff
@@ -0,0 +1,195 @@
+#ifndef Tile_h
+#define Tile_h
+
+class Tile{
+  private:
+    char datos1, datos2;
+
+  public:
+    Tile();
+
+    //getters
```

Comment:
Every comment in the format:
Ex 1.
// Getters.
Ex 2.
// Setters.
Ex 3.
// This is a comment.
Ex 4.
// The following cases may occur:
// Case 1: blabla.
// Case 2: blabla.
// Case 3: blabla.